grails-app/conf/RoleFilters.groovy

```diff
@@ -63,6 +63,18 @@ class RoleFilters {
             'stockMovementItemApi': ['getStockMovementItems']
     ]
 
+    def static neutralActions = [
```

Comment:
The naming here is the only thing I do not like.

Reply:
Changed to authenticatedActions